keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/KeelRepository.kt

```diff
@@ -127,7 +129,7 @@ interface KeelRepository {
   // START ResourceRepository methods
   fun allResources(callback: (ResourceHeader) -> Unit)
 
-  fun getResource(id: String): Resource<out ResourceSpec>
```

Comment:
Why remove the `out` here? Asking because I'm extremely bad at understanding what it does in the first place, so looking to learn :D

Reply:
+1

Reply:
> Asking because I'm extremely bad at understanding what it does in the first place, so looking to learn :D

Same. 😆 

I removed it because the compiler (who knows much more about Kotlin than I do), said it was redundant/unnecessary. 🙂

Reply:
…and it says that because I added `out` variance on the interface a few days ago making it unnecessary to specify it at the use-site.